src/code/app.js

```diff
@@ -106,8 +104,6 @@ function renderApp() {
                           <Route path="/(:level/:mission/:challenge)" component={ChallengeContainerSelector} />
                           <Route path="/(:challengeId)" component={ChallengeContainerSelector} />
                         </Router>
-                        <ModalMessageContainer />
-                        <NotificationContainer />
                       </div>;
```

Comment:
I moved the messages and notifications into the FV container so they're scaled properly